extensions/tiled-assets/README.md

```diff
@@ -3,7 +3,7 @@
 - **Title: Tiled Assets**
 - **Identifier: tiled-assets**
 - **Field Name Prefix: tiles**
-- **Scope: Item**
+- **Scope: Item, Catalog, Collection**
```

Comment:
The scope in the extensions overview table must be updated, too.

In the JSON Schemas for collections and catalogs you must add the "tiled-assets" value to the stac_extensions enum.

Reply:
I'll do that, thanks!

Reply:
Hm, there is no extensions enumeration in the `catalog.json`.

Reply:
Indeed, I'm not 100% sure why we did that. I guess it should look like in the collection.json?! Anyway, so that's nothing we need to change here.